third_party/purge/helpers/varnish_helper.php

```diff
@@ -30,12 +32,20 @@ function send_purge_request($site_url = NULL, $site_port = NULL)
 			$port = 80;
 		}
 		
+		echo $purge_url; 
+		
 		$ch = curl_init();
+		
 		curl_setopt($ch, CURLOPT_URL, $purge_url);
 		curl_setopt($ch, CURLOPT_PORT , (int)$port);
-		curl_setopt($ch, CURLOPT_CUSTOMREQUEST,'EE_PURGE');
+		curl_setopt($ch, CURLOPT_HTTPHEADER, array( 'Host: '.$_SERVER['SERVER_NAME'] ) );
+		curl_setopt($ch, CURLOPT_CUSTOMREQUEST,'PURGE');
```

Comment:
Why go from `EE_PURGE` to `PURGE`? Was nice having the extra specificity in the VCL.